contents/customers/contra.md

```diff
@@ -0,0 +1,54 @@
+
+As a result of spreading functionality across multiple tools, Contra also faced data inconsistencies. Tools such as Segment were helpful for forwarding data between platforms, but also added to the costs and upkeep required. Eventually, Contra decided to consolidate the stack as much as possible — and that’s where PostHog came in. 
+
+### Using feature flags for a ‘slow roll-out’ strategy
+
+Switching to PostHog enabled Contra to consolidate feature flags, session recordings, product analytics and more into a single tool — but it also enabled the company to come together around a single platform as well. Instead of each team in the company relying on a different software, everyone was able to look at a single source of truth.
+
+<BorderWrapper>
+    <Quote
+        imageSource="/images/customers/allison.jpg"
+        size="md"
+        name="Allison Nulty"
+        title="Head of Product, Contra "
+        quote={`“A real positive has been the ability to talk to the PostHog engineers, to share feedback with them and have it received well. I think how rewarding that open line of communication is when I engage with our users too.”`}
+    />
+</BorderWrapper>
+
+“Previously the teams all used different tools,” said Allison. “That led to a lot of confusion, because we were onboarding teams to all these different tools and it was clearly creating complexity and leading to inconsistencies... We had issues with LaunchDarkly especially, and feature flags are very important to us.”
+
+Feature flags in particular are important because the engineering team deploys all new product features behind flags and follows a careful ‘slow roll-out’ strategy. All features are first deployed to beta users, then 20% of all users, followed by 50% of all users and finally 100% of all users. Switching feature flags to PostHog enables Contra to continue testing with this method, while also analyzing feature adoption and performance in the same tool. 
+
+### Using session recordings to improve a product
+
+The combination of feature flags and session recordings has been especially powerful for Contra, as it enables product managers to directly observe how user behavior changes between variants. In other instances, session recording alone enables the team to identify where to make improvements. 
+
+![Contra screenshot](../images/customers/contra/contra_session.gif)
+<Caption>Using PostHog session recording to analyze user sign-ups</Caption>
+
+“We had a waitlist for one feature,” said Allison. “We weren’t seeing expected results, so we used PostHog to look at the funnel and where people were dropping off. We could jump from there straight to the session recordings and that’s when we saw there was a button missing. That meant we were missing a huge number of sign-ups.”
```

Comment:
We were able to easily go from a conversion funel in the analytics tool to session recordings for that dropoff point to identify why users were not converting to the waitlist. With this analysis, we identified one path where users were not seeing a reminder modal and therefore missing out on a large number of waitlist entries.

Reply:
```suggestion
“We used PostHog to look at our funnel for a waitlist in the insights tool,” said Allison. “From there, we could easily jump to session recordings to see the drop-off point. With this analysis, we identified one path where users were not seeing a reminder modal and therefore we were missing out on a large number of waitlist entries.”
```